src/main/java/Duke.java

```diff
@@ -1,10 +1,56 @@
+package main.java;
+
+import java.util.Scanner;
+
 public class Duke {
+    private static Task todoList = new Task(100);
```

Comment:
maybe the todoList can be plural (by adding a 's' behind "List") as it is collection of objects